app/src/createExcelFiles/ExcelUtils.js

```diff
@@ -0,0 +1,23 @@
+import * as XLSX from "xlsx";
+
+export function mapToExcelArray(searchItems) {
+  const result = searchItems.map((searchItem) => [
+    searchItem.title,
+    searchItem.link,
+  ]);
+  result.unshift(["Title", "Links"]);
+  return result;
+}
+
+export function createExcel(googleResults, searchName) {
+  const arrayOfArray = mapToExcelArray(googleResults);
+  let name = searchName;
+  name = name.split(" ").join("");
```

Comment:
I would say it would be better to split and join using `-`
so if the query for `tesla stock` the file should be `tesla-stock.xlsx`
much easier on the eye than reading `teslastock.xlsx`